third-party/eazy/v1/openapi.yaml

```diff
@@ -1435,12 +1469,20 @@ paths:
             type: string
             format: email
             example: 'contactJid@company.com'
-        - in: body
-          name: message
-          description: Color of the label
-          schema:
-            type: integer
-          example: 8
+      requestBody:
+        description: The message you would like to send
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                message:
+                  description: Outgoing message
+                  type: object
+                  example:
+                    body: Hello world
+                    type: text
```

Comment:
Cannot test it. What is the proper fix?

Reply:
I have noticed that you have pushed [some changes to this object](https://github.com/tyntec/api-collection/pull/106/files#diff-fa1b1ded6b1a42d44b72a651240624ebe14243d63d9999faf2dfac83d70a8ed5R1511-R1524) to the master branch already but still... Does the `message` object need to have its properties documented?